modules/services/window-managers/river/options.nix

```diff
@@ -0,0 +1,492 @@
+                };
+              };
+            });
+          description =
+            "Run command when river receives a tablet mode switch event.";
+          example = {
+            normal = {
+              on = "foo";
+              off = "foo";
+            };
+            locked.on = "foo";
+          };
+          default = { };
+        };
+      };
+
+      # CONFIGURATION
+      attachMode = mkOption {
+        type = types.enum [ "top" "bottom" ];
+        default = "bottom";
+        description =
+          "Configure where new views should attach to the view stack.";
+      };
+
+      backgroundColor = mkOption {
+        type = types.str;
+        default = "0x002b36";
+        description = "Background color in rrggbb format.";
+        example = "#ffffff";
+      };
+
+      border = {
+        color = {
+          focused = mkOption {
+            type = types.str;
+            default = "#93a1a1";
+            description = "Focused border color.";
+          };
+
+          unfocused = mkOption {
+            type = types.str;
+            default = "#586e75";
+            description = "Unfocused border color.";
+          };
+
+          urgent = mkOption {
+            type = types.str;
+            default = "#ff0000";
+            description = "Urgent border color.";
+          };
+        };
+
+        width = mkOption {
+          type = types.int;
+          default = 2;
+          description = "Width of the border in pixels.";
+        };
+      };
+
+      focusFollowsCursor = mkOption {
+        type = types.enum [ "disabled" "normal" "always" ];
+        default = "disabled";
+        description = ''
+          Whether to focus views with the mouse cursor.
+
+          There are three available modes:
+          - disabled: Moving the cursor does not affect focus. This is the default.
+          - normal: Moving the cursor over a view will focus that view. Moving the cursor within a view will not re-focus that view if focus has moved elsewhere.
+          - always: Moving the cursor will always focus whatever view is under the cursor.
+
+          If the view to be focused is on an output that does not have focus, focus is switched to that output.
+        '';
+      };
+
+      hideCursor = {
+        timeout = mkOption {
+          type = types.int;
+          default = 0;
+          description = ''
+            Hide the cursor if it wasn't moved in the last timeout milliseconds until it is moved again.
+            The default value is 0, which disables automatically hiding the cursor.
+            Show the cursor again on any movement.
+          '';
+        };
+
+        whenTyping = mkOption {
+          type = types.bool;
+          default = false;
+          description = ''
+            Hide the cursor when pressing any non-modifier key.
+            Show the cursor again on any movement.
+          '';
+        };
+      };
```

Comment:
IMO all these camelCase options are redundant.

There is no advantage having these options documented in Home Manager's options page over the official documentation.